arviz/stats/stats.py

```diff
@@ -484,14 +484,10 @@ def loo(data, pointwise=False, reff=None, scale="deviance"):
 
     """
     inference_data = convert_to_inference_data(data)
-    for group in ("posterior", "sample_stats"):
-        if not hasattr(inference_data, group):
-            raise TypeError(
-                "Must be able to extract a {group} group from data!".format(group=group)
-            )
+    if not hasattr(inference_data, "sample_stats"):
+        raise TypeError("Must be able to extract a sample_stats group from data!")
```

Comment:
Im fine with these except the exclamation marks. I know its picky but i dont think they add anything. Would rather leave the statement as all fact devoid of emotional interpretation

Reply:
I'm happy to remove them, but they were already there and also appear elsewhere even in the same file.

Reply:
In that case we should remove all exclamations marks to be consistent

Reply:
Oops I missed that. Thanks @sethaxen for doing some cleanup and pointing that out.